airbyte-commons/src/main/java/io/airbyte/commons/lang/CloseableShutdownHook.java

```diff
@@ -7,22 +7,21 @@
 import com.google.common.annotations.VisibleForTesting;
 import java.io.Closeable;
 import java.util.Collection;
-import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Registers a shutdown hook that calls the close method of the provided objects. If an object does
  * not support either the {@link AutoCloseable} or {@link Closeable} interface, it will be ignored.
- *
+ * <p>
  * This is a temporary class that is being provided to ensure that resources created by each
  * application are properly closed on shutdown. This logic will no longer be necessary once an
  * application framework is introduced to the project that can provide object lifecycle management.
  */
 public class CloseableShutdownHook {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(CloseableShutdownHook.class);
+  private static final Logger log = LoggerFactory.getLogger(CloseableShutdownHook.class);
```

Comment:
why switch from all caps to all down case?

Reply:
The `@SLF4j` annotation generates a lower case log variable:
```
Example:
  @Slf4j
  public class LogExample {
  }
  
will generate:
  public class LogExample {
      private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LogExample.class);
  }  
```
And I've been stripping out this annotation and replacing it with non-generated `log` code. We presently have a mix of `log` and `LOGGER` and should standardize on one and update everything to match.